gapic/generator/generator.py

```diff
@@ -70,9 +70,14 @@ def get_response(self, api_schema: api.API) -> CodeGeneratorResponse:
         """
         output_files: Dict[str, CodeGeneratorResponse.File] = OrderedDict()
 
+        # TODO: handle sample_templates specially, generate samples.
+        sample_templates, other_templates = utils.partition(
+            self._env.loader.list_templates(),
+            lambda fname: os.path.basename(fname) == "sample.py.j2")
+
         # Iterate over each template and add the appropriate output files
         # based on that template.
-        for template_name in self._env.loader.list_templates():
+        for template_name in other_templates:
```

Comment:
Hanzhen suggested `client_templates`, which I think is good, as is `library_templates`. I think that `gapic_templates` is misleading, since from once perspective _all_ the templates are gapic templates, and `non_gapic_templates` is what an old math teacher of mine would have called "true but not useful."